tests/setupTests.js

```diff
@@ -0,0 +1,4 @@
+import Enzyme from 'enzyme';
```

Comment:
You can destructure configure from enzyme directly. import {configure} from 'enzyme'

Reply:
Used the de-structured version of configure.